packages/valory/services/simple_abci/service.yaml

```diff
@@ -0,0 +1,31 @@
+name: simple_abci
```

Comment:
I'm trying to locally run the service to check it:

```
autonomy init --reset --local
autonomy deploy build deployment valory/simple_abci:bafybeif5infufz5m2ryh26e6vhmh5sqyejeeizarcsbazkktujfnymvcee keys.json
cd abci_build/
docker-compose up --force-recreate
```

And when the agents start I get this:
```
Starting AEA 'agent' in 'async' mode...
[2022-08-04 11:18:44,112] [ERROR] [agent] Error while connecting <class 'connection_module.ABCIServerConnection'>: FileNotFoundError(2, 'No such file or directory')
```

@angrybayblade any idea?